pkg/storaged/vdos-panel.jsx

```diff
@@ -31,9 +31,12 @@ import { dialog_open, TextInput, SelectSpace, SizeSlider, CheckBoxes } from "./d
 const _ = cockpit.gettext;
 
 export function vdo_feature(client) {
+    const os_vdo_map = get_config("vdo_package", false);
+    const os_release = JSON.parse(window.localStorage['os-release'] || "{}");
```

Comment:
My idea was that `get_config` already does all the os-release lookup, and this code here would stay unchanged.  Then we can super cleanly use the same mechanism immediately for "nfs_client_package".

Reply:
Something like:

```
export function get_config(name, def) {
    if (cockpit.manifests.storage && cockpit.manifests.storage.config) {
        var val = cockpit.manifests.storage.config[name];
        if (typeof val === 'object' && val !== null) {
            const os_release = JSON.parse(window.localStorage['os-release'] || "{}");
            val = val[os_release.ID];
        }
        return val !== undefined ? val : def;
    } else {
        return def;
    }
}
```

Reply:
Wdyt?

Reply:
I see! I didn't dare to do that yet before looking at the specifics of NFS, but indeed it makes sense to handle both of them at the same time, to make sure the `get_config()` API works for both of them. Landing this PR isn't super-urgent, so I'll do that first and repost.

Reply:
Done, and added the corresponding thing for NFS. I am still testing that locally, though, I'll ask for re-review when ready.